src/test/resources/application-unittest.properties

```diff
@@ -1,6 +1 @@
 datarepo.testWithEmbeddedDatabase=true
```

Comment:
Why do we default this parameter to true for unit tests?  This feels like something that should default to false and be set to true when we tag a test as an `@EmbeddedDatabaseTest`.

In some tests without embedded databases, we set this to false, but not always. (I'm not reading into our intentions much here, though.)

I tried defaulting it to false to see what happened (lots of test failures), but I didn't have a chance to dig deeper.

Reply:
I'm guessing the thought was that most unit tests needed the embedded dataset, but I think your suggestion of using the `@EmbeddedDatabaseTest` test tag and defaulting to false makes sense. Could be a future change.

Reply:
Just to capture here since we talked about it in standup: I think it's worth investigating in a separate ticket but I might propose looking into what is a bug in our test infra rather than setting the default to false.  Looks like there's some wiring stuff that is happening if we don't include `(properties = {"datarepo.testWithEmbeddedDatabase=false"})` with the `@SpringBootTest` annotation.  That feels odd (it may very well mean flipping the default but I want to make sure that's actually the right thing to do)

Reply:
Thanks for capturing that nuance, I agree!  My instinct is that there are side effects of this property that we don't fully understand yet, and possibly we'll want to decouple those side effects from the prop rather than simply switching the default.  I will file a ticket.

Reply:
https://broadworkbench.atlassian.net/browse/DR-3073